server/gitrest/Dockerfile

```diff
@@ -3,6 +3,8 @@
 
 FROM node:12.18-stretch AS base
 
+ENV EXTERNAL_STORAGE_ENABLED = false
```

Comment:
Should the spaces around the equal sign be removed?
https://docs.docker.com/engine/reference/builder/#env